config.js

```diff
@@ -0,0 +1,2 @@
+
+export const BASE_URL = 'https://vcoach-eoymq.ondigitalocean.app/vcoach-backend2/api/v1'
```

Comment:
Didn't we include the base url in the `.env` file, if yes, please remove this and add `BASE_URL=` to the `env.template` file